src/components/Login.js

```diff
@@ -79,84 +75,90 @@ class Login extends React.Component {
           x1="547"
           y1="0"
           x2="547"
-          y2={`${window.innerHeight}`}
+          y2={`${windowHeight}`}
           gradientUnits="userSpaceOnUse"
         >
-          <stop stopColor={gradientColors[this.state.index][0]} />
-          <stop offset="1" stopColor={gradientColors[this.state.index][1]} stopOpacity="0.47" />
+          <stop stopColor={gradientPrimary} />
+          <stop offset="1" stopColor={gradientSecondary} stopOpacity="0.47" />
         </linearGradient>
       </defs>
     </svg>
   );
+  // basically, when the window resizes, recalculates getBackgroundSVG - the window parameters change!
+  useEffect(() => {
+    window.addEventListener('resize', () => {
+      setWindowHeight(window.innerHeight);
+    });
+    return () => window.removeEventListener('resize', () => setWindowHeight(window.innerHeight));
```

Comment:
Is this the proper way to remove the event listener? I'm realizing that I may have made a mistake when I wrote the code initially. The two different instances of `() => setWindowHeight(window.innerHeight)` are anonymous functions with different underlying references, right? 

If that's the case, this code just needs to look like:

```suggestion
    const resizeHandler = setWindowHeight(window.innerHeight);
    window.addEventListener('resize', resizeHandler);
    return () => window.removeEventListener('resize', resizeHandler);
```

Reply:
Other than this, I'm good to merge. Not necessary to wait on me to approve as well, since Timothy's already given it the green light!

Reply:
This does the trick in getting the SVG to update in the intended way but there is a buggy side effect that I've experienced before - If the page is rendered and the window height is increased, you get a buggy looking SVG like the one in the screencap:
<img width="841" alt="Screen Shot 2021-11-11 at 7 48 07 PM" src="https://user-images.githubusercontent.com/18103497/141389986-e36743c1-b1b2-447b-a2b2-d4880d722ac0.png">

With this code tweak:
```
const resizeHandler = () => setWindowHeight(window.innerHeight);
    window.addEventListener('resize', resizeHandler);
    return () => window.removeEventListener('resize', resizeHandler); 
```
It removes the bug, but the SVG doesn't have the cool swooping effect.